src/researches/keyphraseDistribution.js

```diff
@@ -147,33 +148,30 @@ const step = function( maximizedSentenceScores, stepSize ) {
  * Computes the punishment for not using all content words within the window. Windows are determined in the same way as
  * in the step function
  *
- * @param {Array} sentences The sentences to analyze.
- * @param {Object} topicForms The topicForms of the paper.
- * @param {string} locale The locale of the paper.
- * @param {number} stepSize The number of sentences that should be in every step to average over.
+ * @param {Array} sentenceScores The array of sentenceScores to calculate penalty based on.
  *
- * @returns {Array} The scores per portion of text.
+ * @returns {number} The additional penalty for not using all keyphrase terms.
  */
```

Comment:
Function does not use a window anymore, so the function description can be adapted to reflect that.